### PR TITLE
[codestyle] removed unnecessary constructor

### PR DESCRIPTION
**Pull request by akshitdewan**

### CODE REVIEW

File: xwiki-rendering-api/src/main/java/org/xwiki/rendering/block/SpaceBlock.java [hunk 1]
```diff
@@ -36,10 +36,6 @@
      *
      * @since 2.6RC1
      */
```
Comment (tmortagne) [inline xwiki-rendering-api/src/main/java/org/xwiki/rendering/block/SpaceBlock.java]: You need to remove the javadoc of the constructor too.

**Comment by akshitdewan** [comment]
Thomas, I removed the part you highlighted, not sure if you want to remove whats above that because it seems like an important description of a SpaceBlock.

**Comment by tmortagne** [comment]
This was is the constructor javadoc so it does not make any sense to keep it there. You should move the part related to SpaceBlock in the class javadoc.

**Comment by akshitdewan** [comment]
Moved them

**Comment by tmortagne** [comment]
Thanks for your contribution !

**Pull request merged**